dask_expr/_collection.py

```diff
@@ -413,6 +414,12 @@ def compute(self, fuse=True, **kwargs):
         out = out.optimize(fuse=fuse)
         return DaskMethodsMixin.compute(out, **kwargs)
 
+    def explain(self, stage: OptimizerStage = "fused", format: str | None = None):
```

Comment:
nit: a custom file name would be nice as an argument, wouldn't it?

Reply:
Stop bloating my very dumb initial version! :D You're right, though.